pyswarms/single/local_best.py

```diff
@@ -172,7 +172,7 @@ def __init__(
         # Initialize the resettable attributes
         self.reset()
         # Initialize the topology
-        self.top = Ring()
+        self.top = Ring(static=False)
```

Comment:
Keep this so that we are explicit :+1:

Reply:
Hi @whzup , you can also add a `cli_print` somewhere here so that they know about it.
I forgot to tell you to add a `static` parameter here. We can add control to `LocalBestPSO` by letting the user to choose between a static or dynamic `Ring` topology. :+1: